### PR TITLE
Create c001.mdx

### LINKED ISSUE #95: Create a page for `c001`
### description
Make `/docs/upgrades/locks/c001.mdx` using the [lock template](https://github.com/comcode-org/hackmud_wiki/blob/main/templates/lock.mdx) as a starter. Write a page about the lock that complies with the [content guide](https://github.com/comcode-org/hackmud_wiki?tab=readme-ov-file#content-guide).

### PR DESCRIPTION
**Pull request by kbinreallife**
fixes #95

### CODE REVIEW

File: docs/upgrades/locks/c001.mdx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+```
+LOCK_ERROR
+<input> is not the correct color name.
+```
+
+## Solving
+
+<details>
+<summary>Spoilers for c002's solutions</summary>
```
Comment (Fayti1703) [inline docs/upgrades/locks/c001.mdx:30]: Incorrect lock name:
```suggestion
<summary>Spoilers for c001's solutions</summary>
```

File: docs/upgrades/locks/c001.mdx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+### c002
```
Comment (Fayti1703) [inline docs/upgrades/locks/c001.mdx:32]:
```suggestion
### c001
```

File: docs/upgrades/locks/c001.mdx [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+c002 will choose a random color from a list of the following 8 colors, as a string, for its solution:
```
Comment (Fayti1703) [inline docs/upgrades/locks/c001.mdx:34]:
```suggestion
c001 will choose a random color from a list of the following 8 colors, as a string, for its solution:
```

**Review by Fayti1703 — CHANGES_REQUESTED** [review]

**Review by Fayti1703 — APPROVED** [review]

File: docs/upgrades/locks/c001.mdx [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+When providing any truthy argument (or an empty string `""`), the lock will state that the correct color was not provided, unless the correct color is provided:
+
+```
+>>user.fakeloc123 { c001: "not_a_color" }
+LOCK_ERROR
+not_a_color is not the correct color name.
+```
+
+## Solving
```
Comment (seanmakesgames) [inline docs/upgrades/locks/c001.mdx:29]: We are missing the `color_digit` bits

**Review by seanmakesgames — CHANGES_REQUESTED** [review]

**Review by seanmakesgames — APPROVED** [review]
mucho pairing

**Pull request merged**